discounts/wasm/delivery-discount-types/default/input.graphql

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Nit: empty queries are actually invalid and will fail to push. I'd suggest putting at least 1 field.

Reply:
Could this be a feature though? We want script authors to be judicious with their input queries, so if pushing without modifying `input.graphql` causes an error, then they have to put some thought into it?

Reply:
+1 would prefer being able to have an empty input query 😛